### DISCUSSION TITLE
How to set MTU of NIC in guest OS by cloud-hypervisor options?

### DISCUSSION THREAD

**Discussion by carezkh** [cloud-hypervisor/cloud-hypervisor#4491] [Q&A] [answered]
Hi! How to set MTU of NIC in guest OS by cloud-hypervisor options?

**Comment by liuw** [comment, marked as answer]
There is no such an option yet.

Patches are welcome. You will need to use `VIRTIO_NET_F_MTU` to express it to the guest I suppose.

**Reply by carezkh** [reply]
Thanks! I will give it a try.
